plugins/qtosgrave/osglodlabel.cpp

```diff
@@ -24,14 +33,18 @@ OSGLODLabel::OSGLODLabel(const std::string& label) : osg::LOD() {
     *     [Label Text]
     */
 
-    // Set up text element and its properties
+    // Create text element
     osg::ref_ptr<osgText::Text> text = new osgText::Text();
+    // Read copy QT resource to temp location and stream that into OSG to use
+    std::string tmp = std::tmpnam(NULL);
+    QFile::copy(":/fonts/NotoSans-Regular.ttf", QString::fromStdString(tmp));
```

Comment:
Hey Jonathan, a hint here, you can read the resource as a bytearray:
`QByteArray ba = file.readAll();`
Then you can just do `ba.toStdString()`

Reply:
I dont think you need to copy the file to another filepath. Its not recommended as well. We dont know where the openrave is running (can even be in readonly filesystem). Better just use memory.

Reply:
I tried earlier to convert `file.readAll()` into a bytearray before taking its data and putting it into an `istream` and having OSG read it that way, but the string kept failing to read correctly. I can try seeing if `ba.toStdString()` gets around that issue.

Reply:
And probably let the Qt part take care loading font stream, then LODLabel does not need to bother with where to load Qt resource.

Reply:
You might need to open the file as well ```open(QIODevice.ReadOnly | QIODevice.Unbuffered)```